tests/test_endpoint.php

```diff
@@ -199,6 +202,15 @@ public function test_create_post_without_create_scope() {
 		static::$scopes = array( 'post' );
 	}
 
+	public function test_create_post_subscriber_id() {
+		static::$scopes = array( 'create' );
+		$response       = $this->dispatch( self::create_form_request( static::$post ), static::$subscriber_id );
+		self::check( $response, 403, sprintf( 'user id %1$s cannot create posts', static::$subscriber_id ) );
+		// Set Back to Default
```

Comment:
minor: a better way to do this is in `setUp()`: https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#shared-setup-between-related-tests . more reliable. (same w/lines 201-202, and maybe others.)